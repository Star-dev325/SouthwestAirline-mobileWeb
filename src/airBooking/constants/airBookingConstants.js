export const ANALYTICS = {
  FLIGHT_CONFIRMATION_PAGE: {
    page: 'air-booking-confirmation',
    page_channel: 'BOOK',
    page_name: 'confirmation',
    page_subchannel: 'SWA'
  },
  frequentTraveler: {
    page_channel: "BOOK",
    page_name: 'frequent traveler',
    page_subchannel: "SWA"
  },
  MULTI_SELECT_INBOUND_FARE_PAGE: {
    page: 'air-booking-select-multi-inbound-fare',
    selectpageindex: '1'
  },
  MULTI_SELECT_INBOUND_PAGE: {
    page: 'air-booking-select-multi-inbound',
    selectpageindex: '1'
  },
  MULTI_SELECT_OUTBOUND_BOUNDS_PAGE: {
    page: 'air-booking-select-multi-outbound',
    selectpageindex: '0'
  },
  MULTI_SELECT_OUTBOUND_FARE_PAGE: {
    page: 'air-booking-select-multi-outbound-fare',
    selectpageindex: '0'
  },
  MULTI_SELECT_OUTBOUND_PAGE: {
    page: 'air-booking-select-multi',
    selectpageindex: '0'
  },
  MULTI_SELECT_PAGE_CONSTANTS: {
    page_channel: 'air',
    page_name: 'shopping',
    page_subchannel: 'booking'
  },
  PRICE_PAGE: {
    page: 'air-booking-pricing-summary',
    page_channel: 'BOOK',
    page_name: 'Price Page',
    page_subchannel: 'SWA'
  },
  PURCHASE_PAGE_LOAD: 'air booking purchase page',
  YOUNG_TRAVELER_PAGE: {
    page: 'air-booking-young-traveler',
    page_channel: 'BOOK',
    page_name: 'young-traveler',
    page_subchannel: 'SWA'
  }
};
export const CHASE = {
  PAYMENT_FAILURE_ERROR_CODE: 404519017
};
export const CHASE_AUTO_PROVISIONING = {
  CHASE_CARD_PROVISION: 'CHASE_CARD_PROVISION',
  CHASE_EMAIL_PROVISION: 'CHASE_EMAIL_PROVISION'
};
export const TRAVEL_FUNDS = {
  TOKEN_EXPIRED_AIRBOOKING_CODE: 400310756
};
export const SPLIT_PAY_CALC_FUNDS_HREF = '/v1/mobile-air-booking/page/x-calculate-funds/SPLIT_PAYMENT';
