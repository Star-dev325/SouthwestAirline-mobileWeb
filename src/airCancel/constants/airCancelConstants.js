export const AIR_CANCEL_FLOW_NAME = 'airCancel';
export const AIR_CANCEL_SPLIT_PNR_FLOW_NAME = 'airCancelSplitPnr';
export const ANALYTICS = {
  AIR_CANCEL_CONFIRMATION_PAGE: {
    page: 'air-cancel-refund-summary'
  },
  AIR_CANCEL_REVIEW_PAGE: {
    page: 'air-cancel'
  },
  AIR_CANCEL_SELECT_BOUND_PAGE: {
    page: 'air-cancel-select-bound',
    page_channel: 'air',
    page_name: 'select-bound',
    page_subchannel: 'cancel'
  },
  AIR_CANCEL_SELECT_PASSENGERS_PAGE: {
    page: 'air-cancel-select-passengers',
    page_channel: 'air',
    page_name: 'select-passengers',
    page_subchannel: 'cancel'
  }
};
export const CANCEL_SPLIT_PNR_CONFIRMATION = 'CANCEL_SPLIT_PNR_CONFIRMATION';
export const ERRORS = {
  CANCEL_BOUND_REFUND_QUOTE_MISMATCH: 400620560
};
