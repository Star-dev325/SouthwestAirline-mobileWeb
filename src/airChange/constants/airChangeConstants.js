export const DEPARTURE = 'Departure';
export const RETURN = 'Return';
export const REFUND_SUMMARY = {
  TOTAL: 'TOTAL'
};
export const CONFIRMATION = {
  TOTAL: 'TOTAL',
  REFUND_SUMMARY: {
    TRAVEL_FUNDS: 'TRAVEL FUNDS'
  }
};
export const SELECTION_MODE = {
  ALL: 'ALL',
  SINGLE: 'SINGLE',
  NONE: 'NONE'
};
export const ZERO_POINTS = {
  fare: {
    amount: '0',
    currencyCode: 'PTS',
    currencySymbol: ''
  }
};
export const ZERO_DOLLARS = {
  fare: {
    amount: '0.00',
    currencyCode: 'USD',
    currencySymbol: '$'
  }
};
export const REACCOM_VIEW_RESERVATION_MESSAGE = {
  REACCOM_CONTACT_US_TO_CHANGE_FLIGHT_FREE: 'REACCOM_CONTACT_US_TO_CHANGE_FLIGHT_FREE',
  REACCOM_VIEW_RESERVATION: 'REACCOM_VIEW_RESERVATION',
  REACCOM_VIEW_RESERVATION_FLIX: 'REACCOM_VIEW_RESERVATION_FLIX',
  REACCOM_VIEW_RESERVATION_GDS: 'REACCOM_VIEW_RESERVATION_GDS'
};
export const FLIGHT_PRODUCT_TYPE = {
  DYNAMIC_WAIVER: 'DYNAMIC_WAIVER',
  NORMAL: 'NORMAL'
};
export const FLIGHT_TYPE = {
  DEPARTURE: 'DEPARTURE',
  RETURN: 'RETURN'
};
export const TRAVEL_FUNDS = {
  TOKEN_EXPIRED_AIRCHANGE_CODE: 400310756
};

export const ANALYTICS = {
  CONFIRMATION_PAGE: {
    page: 'air-change-confirmation'
  },
  REVIEW_PAGE: {
    page: 'air-change-pricing-review'
  },
  SELECT_PASSENGERS_PAGE: {
    extra_seat: '0',
    page: 'air-change-select-passengers',
    page_channel: 'air',
    page_name: 'select-passengers',
    page_subchannel: 'change'
  },
  SELECT_BOUND_PAGE: {
    page: 'air-change-select-bound',
    page_channel: 'air',
    page_name: 'select-bound',
    page_subchannel: 'change'
  },
  SHOPPING_PAGE: {
    page: 'air-change-shopping',
    page_channel: 'air',
    page_name: 'shopping',
    page_subchannel: 'change'
  },
  REACCOM_CONFIRMATION_PAGE: {
    formcomplete: '1',
    formname: 'reaccom change',
    page: 'air-reaccom-confirmation',
    page_channel: 'air',
    page_name: 'confirmation',
    page_subchannel: 'reaccom'
  },
  REACCOM_SELECT_FLIGHT_PAGE: {
    page: 'air-reaccom-view',
    page_channel: 'air',
    page_name: 'view',
    page_subchannel: 'reaccom'
  },
  REACCOM_INBOUND_SHOPPING_PAGE: {
    page: 'air-reaccom-select-return',
    page_channel: 'air',
    page_name: 'select-return',
    page_subchannel: 'reaccom'
  },
  REACCOM_OUTBOUND_SHOPPING_PAGE: {
    page: 'air-reaccom-select',
    page_channel: 'air',
    page_name: 'select',
    page_subchannel: 'reaccom'
  },
  REACCOM_SUMMARY_PAGE: {
    page: 'air-reaccom-summary',
    page_channel: 'air',
    page_name: 'summary',
    page_subchannel: 'reaccom'
  }
};
export const AIR_CHANGE_ARD_CODE = 403500291;
export const AIR_CHANGE_SPLIT_PNR_FLOW_NAME = 'airChangeSplitPnr';
