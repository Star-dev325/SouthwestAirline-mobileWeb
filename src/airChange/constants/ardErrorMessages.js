export default {
  TITLE: 'Under Construction',
  MESSAGE:
    "Our Reservation System is undergoing some changes and we can't book this particular itinerary through the mobile app until the dust settles. To complete the booking visit Southwest.com."
};
