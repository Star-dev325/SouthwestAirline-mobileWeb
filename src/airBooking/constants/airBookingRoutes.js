import { carBookingOldRoutes, carBookingRoutes } from 'src/carBooking/constants/carBookingRoutes';
import { earlyBirdOldRoutes, earlyBirdRoutes } from 'src/earlyBird/constants/earlyBirdRoutes';

export const airBookingRoutes = {
  addManualIrn: '/air/booking/manual-irn',
  applyRapidRewards: '/air/booking/apply-rapid-rewards?clearFormData=false',
  applyTravelFunds: '/air/booking/apply-travel-funds',
  billingAddress: '/air/booking/billing-address',
  carBookingIndex: carBookingRoutes['index'],
  checkin: earlyBirdRoutes['checkin'],
  confirmation: '/air/booking/confirmation.html',
  contactInfoTravelManager: '/air/booking/contact-info-travel-manager',
  contactMethod: '/air/booking/contact-method',
  fareDetails: '/air/booking/fare-details',
  flightShoppingDepart: '/air/booking/select-depart.html',
  flightShoppingReturn: '/air/booking/select-return.html',
  flightShoppingOutbound: '/air/booking/outbound/results',
  frequentTravelers: '/air/booking/passenger/:paxNumber/frequent-travelers',
  index: {
    canonicalPath: '/air/booking/',
    htmlPath: '/air/booking/index.html'
  },
  indexWithoutClearForm: '/air/booking?clearFormData=false',
  internalReferenceNumber: '/air/booking/irn-info',
  lowFareCalendar: {
    canonicalPath: '/air/low-fare-calendar/',
    htmlPath: '/air/low-fare-calendar/select-dates.html'
  },
  lowFareCalendarDate: '/air/low-fare-calendar/date-select',
  passengers: '/air/booking/passenger',
  passengersWithPassport: '/air/booking/passenger/:paxNumber/passport',
  passengersWithPax: '/air/booking/passenger/:paxNumber',
  passengersWithPaxEdit: '/air/booking/passenger/:paxNumber/edit',
  passengersWithSpecialAssistance: '/air/booking/passenger/:paxNumber/special-assistance',
  payment: '/air/booking/payment',
  paymentEdit: '/air/booking/payment/edit',
  price: '/air/booking/price.html',
  priceReview: '/air/booking/price/detail',
  purchase: '/air/booking/purchase.html',
  purchasePaypal: '/air/booking/purchase/paypal',
  purchasePaypalCanceled: '/air/booking/purchase/paypal-canceled',
  purchaseWithoutClearForm: '/air/booking/purchase.html?clearFormData=false',
  recent: '/air/booking/recent',
  reprice: '/air/booking/reprice',
  selectCompany: '/air/booking/select-company',
  selectCompanyToggle: '/air/booking/select-company?CLK=swabizomnitoggle',
  selectDepartFare: '/air/booking/select-fare-depart.html',
  selectFare: '/air/booking/:direction/select-fare',
  selectPassengers: '/air/booking/select-passenger',
  selectReturnFare: '/air/booking/select-fare-return.html',
  viewReservationView: '/air/manage-reservation/view/:recordLocator',
  youngTraveler: '/air/booking/young-traveler',
  youngTravelerEditWithoutClearForm: '/air/booking/young-traveler/edit?clearFormData=false',
  youngTravelerParentConsent: '/air/booking/young-traveler-parent-consent'
};

export const airBookingOldRoutes = {
  addManualIrn: '/air/booking/addManualIrn',
  applyRapidRewards: '/air/booking/apply-rapid-rewards?clearFormData=false',
  carBookingIndex: carBookingOldRoutes['index'],
  checkin: earlyBirdOldRoutes['checkin'],
  confirmation: '/air/booking/confirmation',
  fareDetails: '/air/booking/shopping/fare-details',
  flightShoppingDepart: '/air/booking/shopping/:paxType/:direction/results',
  flightShoppingReturn: '/air/booking/shopping/:paxType/:direction/results',
  flightShoppingOutbound: '/air/booking/shopping/adult/outbound/results',
  frequentTravelers: '/air/booking/passengers/:paxNumber/frequent-travelers',
  index: '/air/booking/shopping',
  indexWithoutClearForm: '/air/booking/shopping?clearFormData=false',
  internalReferenceNumber: '/air/booking/irnInfo',
  lowFareCalendar: '/air/booking/shopping/low-fare-calendar',
  lowFareCalendarDate: '/air/booking/shopping/low-fare-calendar/date-select',
  passengers: '/air/booking/passengers',
  passengersWithPassport: '/air/booking/passengers/:paxNumber/passport',
  passengersWithPax: '/air/booking/passenger/:paxNumbers',
  passengersWithPaxEdit: '/air/booking/passengers/:paxNumber/edit',
  passengersWithSpecialAssistance: '/air/booking/passengers/:paxNumber/special-assistance',
  price: '/air/booking/pricing/summary',
  priceReview: '/air/booking/pricing/review',
  purchase: '/air/booking/review',
  purchaseWithoutClearForm: '/air/booking/review?clearFormData=false',
  recent: '/air/booking/shopping/recent',
  reprice: '/air/booking/pricing/repricing',
  selectCompany: '/air/booking/shopping/select-company',
  selectCompanyToggle: '/air/booking/shopping/select-company?CLK=swabizomnitoggle',
  selectDepartFare: '/air/booking/shopping/:paxType/:direction/select-fare',
  selectFare: '/air/booking/shopping/:paxType/:direction/select-fare',
  selectPassengers: '/air/booking/shopping/select-passengers',
  selectReturnFare: '/air/booking/shopping/:paxType/:direction/select-fare',
  viewReservationView: '/view-reservation/trip-details/:recordLocator',
  youngTraveler: '/air/booking/young-traveler',
  youngTravelerEditWithoutClearForm: '/air/booking/young-traveler/edit?clearFormData=false',
  youngTravelerParentConsent: '/air/booking/young-traveler-parent-consent'
};
