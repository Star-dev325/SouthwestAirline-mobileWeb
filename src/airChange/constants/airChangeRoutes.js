import { carBookingOldRoutes, carBookingRoutes } from 'src/carBooking/constants/carBookingRoutes';
import { travelFundsOldRoutes, travelFundsRoutes } from 'src/travelFunds/constants/travelFundsRoutes';
import { viewReservationOldRoutes, viewReservationRoutes } from 'src/viewReservation/constants/viewReservationRoutes';

export const airChangeRoutes = {
  applyTravelFunds: '/air/change/apply-travel-funds',
  carBookingIndex: carBookingRoutes['index'],
  confirmation: '/air/change/confirmation.html',
  contactMethod: '/air/change/contact-method',
  fareDetails: '/air/change/fare-details',
  flightShopping: '/air/change/:direction/results',
  flightShoppingIndex: '/air/change/find-flights.html',
  payment: '/air/change/payment',
  price: '/air/change/price.html',
  purchasePaypal: '/air/change/reconcile/paypal',
  purchasePaypalCanceled: '/air/change/reconcile/paypal-canceled',
  reaccom: '/air/change/reaccom/details',
  reconcile: '/air/change/reconcile.html',
  reprice: '/air/change/reprice',
  selectFare: '/air/change/:direction/select-fare',
  selectPassengers: '/air/change/select-passengers.html',
  travelFundsIndex: travelFundsRoutes['index'],
  view: {
    canonicalPath: '/air/change/',
    htmlPath: '/air/change/view.html'
  },
  viewReservationIndex: viewReservationRoutes['index']
};

export const airChangeOldRoutes = {
  applyTravelFunds: '/air/change/apply-travel-funds',
  carBookingIndex: carBookingOldRoutes['index'],
  confirmation: '/air/change/confirmation',
  contactMethod: '/air/change/contact-method',
  fareDetails: '/air/change/shopping/fare-details',
  flightShopping: '/air/change/shopping/:paxType/:direction/results',
  flightShoppingIndex: '/air/change/shopping',
  payment: '/air/change/pricing/payment',
  price: '/air/change/pricing/summary',
  purchasePaypal: '/air/change/pricing/review/paypal',
  purchasePaypalCanceled: '/air/change/pricing/review/paypal-canceled',
  reaccom: '/air/change/reaccom/summary',
  reconcile: '/air/change/pricing/review',
  reprice: '/air/change/pricing/repricing',
  selectFare: '/air/change/shopping/:paxType/:direction/select-fare',
  selectPassengers: '/air/change/select-passengers',
  travelFundsIndex: travelFundsOldRoutes['index'],
  view: '/air/change',
  viewReservationIndex: viewReservationOldRoutes['index']
};
