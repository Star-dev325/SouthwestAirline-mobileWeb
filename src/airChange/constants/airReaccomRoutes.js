import { viewReservationOldRoutes, viewReservationRoutes } from 'src/viewReservation/constants/viewReservationRoutes';

export const airReaccomRoutes = {
  confirmation: '/air/reaccom/confirmation.html',
  flightShopping: '/air/reaccom/:direction/results',
  flightShoppingIndex: '/air/reaccom/find-flights.html',
  reaccom: '/air/reaccom/details',
  view: '/air/reaccom/view.html',
  viewReservationIndex: viewReservationRoutes['index']
};

export const airReaccomOldRoutes = {
  confirmation: '/air/change/confirmation',
  flightShopping: '/air/change/shopping/:paxType/:direction/results',
  flightShoppingIndex: '/air/change/shopping',
  reaccom: '/air/change/reaccom/summary',
  view: '/air/change',
  viewReservationIndex: viewReservationOldRoutes['index']
};
