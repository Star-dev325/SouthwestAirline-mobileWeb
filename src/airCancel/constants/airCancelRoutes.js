import { airBookingRoutes } from 'src/airBooking/constants/airBookingRoutes';
import { checkInOldRoutes, checkInRoutes } from 'src/checkIn/constants/checkInRoutes';
import { travelFundsOldRoutes, travelFundsRoutes } from 'src/travelFunds/constants/travelFundsRoutes';
import { viewReservationOldRoutes, viewReservationRoutes } from 'src/viewReservation/constants/viewReservationRoutes';

export const airCancelRoutes = {
  airBookingIndex: airBookingRoutes['index'],
  airCancelRefundQuote: '/air/cancel-reservation/refund-quote.html',
  checkInAdditionalPassportInformation: checkInRoutes['checkInAdditionalPassportInformation'],
  checkInAdditionalPassportInformationDestination: checkInRoutes['checkInAdditionalPassportInformationDestination'],
  checkInAdditionalPassportInformationGreenCard: checkInRoutes['checkInAdditionalPassportInformationGreenCard'],
  checkInAdditionalPassportInformationVisa: checkInRoutes['checkInAdditionalPassportInformationVisa'],
  checkInBoardingPass: checkInRoutes['checkInBoardingPass'],
  checkInBoardingPosition: checkInRoutes['checkInBoardingPosition'],
  checkInConfirmation: checkInRoutes['checkInConfirmation'],
  checkInIndex: checkInRoutes['checkInIndex'],
  checkInPassportInformation: checkInRoutes['checkInPassportInformation'],
  refundSummary: '/air/cancel-reservation/summary.html',
  selectBound: {
    canonicalPath: '/air/cancel-reservation/',
    htmlPath: '/air/cancel-reservation/view.html'
  },
  selectPassengers: '/air/cancel-reservation/select-passengers.html',
  viewReservationIndex: viewReservationRoutes['index'],
  travelFundsIndex: travelFundsRoutes['index']
};

export const airCancelOldRoutes = {
  airBookingIndex: '/air/booking/shopping',
  airCancelRefundQuote: '/air/cancel/:recordLocator',
  checkInAdditionalPassportInformation: checkInOldRoutes['checkInAdditionalPassportInformation'],
  checkInAdditionalPassportInformationDestination: checkInOldRoutes['checkInAdditionalPassportInformationDestination'],
  checkInAdditionalPassportInformationGreenCard: checkInOldRoutes['checkInAdditionalPassportInformationGreenCard'],
  checkInAdditionalPassportInformationVisa: checkInOldRoutes['checkInAdditionalPassportInformationVisa'],
  checkInBoardingPass: checkInOldRoutes['checkInBoardingPass'],
  checkInBoardingPosition: checkInOldRoutes['checkInBoardingPosition'],
  checkInConfirmation: checkInOldRoutes['checkInConfirmation'],
  checkInIndex: checkInOldRoutes['checkInIndex'],
  checkInPassportInformation: checkInOldRoutes['checkInPassportInformation'],
  refundSummary: '/air/cancel/:recordLocator/refund-summary',
  selectBound: '/air/cancel/:recordLocator/select-bound',
  selectPassengers: '/air/cancel/select-passengers',
  travelFundsIndex: travelFundsOldRoutes['index'],
  viewReservationIndex: viewReservationOldRoutes['index']
};
