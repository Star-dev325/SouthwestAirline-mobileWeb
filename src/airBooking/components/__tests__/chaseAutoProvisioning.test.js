import '@testing-library/jest-dom/extend-expect';
import React from 'react';
import { render } from '@testing-library/react';
import ChaseAutoProvisioning from 'src/airBooking/components/chase/chaseAutoProvisioning.jsx';
import { CHASE_AUTO_PROVISIONING } from 'src/airBooking/constants/airBookingConstants';

describe('chaseAutoProvisioning', () => {
  it('should render when cardOrEmail is CHASE_CARD_PROVISION', () => {
    const myPromoCodePageComponent = createComponent({ cardOrEmail: CHASE_AUTO_PROVISIONING.CHASE_CARD_PROVISION });

    expect(myPromoCodePageComponent.baseElement).toMatchSnapshot();
  });

  it('should render when cardOrEmail is CHASE_EMAIL_PROVISION', () => {
    const myPromoCodePageComponent = createComponent({ cardOrEmail: CHASE_AUTO_PROVISIONING.CHASE_EMAIL_PROVISION });

    expect(myPromoCodePageComponent.baseElement).toMatchSnapshot();
  });
  const createComponent = (props = {}) => {
    const defaultProps = {
      ...props,
      header: 'example1',
      body: 'example2'
    };

    return render(<ChaseAutoProvisioning {...defaultProps} />);
  };
});
