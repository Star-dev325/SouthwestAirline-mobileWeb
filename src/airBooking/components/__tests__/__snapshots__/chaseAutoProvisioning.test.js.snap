// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`chaseAutoProvisioning should render when cardOrEmail is CHASE_CARD_PROVISION 1`] = `
<body>
  <div>
    <div
      class="chase-auto-provisioning"
    >
      <img
        src="/content/mkt/images/landing_pages/chase/chase-credit-card.svg"
      />
      <div
        class="chase--text-container"
      >
        <span
          class="chase--header"
        >
          example1
        </span>
        <div
          class="chase--body"
        >
          example2
        </div>
      </div>
    </div>
  </div>
</body>
`;

exports[`chaseAutoProvisioning should render when cardOrEmail is CHASE_EMAIL_PROVISION 1`] = `
<body>
  <div>
    <div
      class="chase-auto-provisioning"
    >
      <img
        src="/content/mkt/images/landing_pages/chase/chase-envelope.svg"
      />
      <div
        class="chase--text-container"
      >
        <span
          class="chase--header"
        >
          example1
        </span>
        <div
          class="chase--body"
        >
          example2
        </div>
      </div>
    </div>
  </div>
</body>
`;
